### ISSUE TITLE
Raids with same IDs

### ISSUE THREAD

**Issue by Geigerkind** [Geigerkind/Legacy-Logs-Bugtracker#2]
Just look at that log http://legacy-logs.com/Vanilla/Raids/Evaluation/index.php?rid=1797

i downloaded logs first, and everything was fine, but than ally's raid was downloaded with same ID

@Imported

**Comment by hunterdave** [comment]
same problem here

https://legacy-logs.com/TBC/Raids/Evaluation/index.php?rid=238

**Comment by Geigerkind** [comment]
Yep it is super important, I am super sorry that I didn't find time to issue this.

cheers and thanks! :)

**Comment by hunterdave** [comment]
Is it possible to prevent this bug? Do you have any advice for me?

**Comment by Geigerkind** [comment]
Problem on B2B is as it seems that there is some kind of cross faction raiding as far as I know,
so the only possible way is to fix it at the first incomming faction because it has to be some faction. Or to have 2 guilds of the same kind. 

This system is not designed for crossfaction servers at all.
